cli/cmd/options.go

```diff
@@ -428,6 +428,7 @@ func makeProxyFlags(defaults *l5dcharts.Values) ([]flag.Flag, *pflag.FlagSet) {
 
 	proxyFlags.MarkDeprecated("proxy-memory", "use --proxy-memory-request instead")
 	proxyFlags.MarkDeprecated("proxy-cpu", "use --proxy-cpu-request instead")
+	proxyFlags.MarkDeprecated("proxy-version", "use --set instead")
```

Comment:
Was trying to follow the existing format. Personally, I'd want a format like `--set proxy.image.version` that told me how to use the new flag instead of the old one I just tried to use, but I'd find it weird that one deprecated flag has some instructions but the others don't. Shall I modify the description of the others as well?

Reply:
I think the other flags (e.g `proxy-memory`, `proxy-cpu`) don't really depend on helm values which is why they have a different format. I'm not sure off the top of my head if we have any other deprecated flags but I'd go with what feels right for you here :)

Reply:
Got it. Haven't posted any updates recently as I'm still setting up remote containers (not a linkerd issue, an internet issue), hoping to push a commit sooner than later :)